### ISSUE TITLE
Compilation error on `dev` branch

### ISSUE THREAD

**Issue by sayerhs** [OpenFAST/openfast#24]
With the latest updates on the `dev` branch, I get a compilation error when using `GNU gfortran 6.3.0`. I am not sure where the character length 11 is showing up, every entry in `ParamUnitsAry` is neatly formatted and looks like it is 10 characters long. 

```
openfast/modules-local/aerodyn/src/AeroDyn_IO.f90:3092:122:

                                "(N/m)     ","(N/m)     ","(N/m)     ","(N/m)     ","(N/m)     ","(N/m)     ","(-)       ", &
                                                                                                                          1
Error: Different CHARACTER lengths (10/11) in array constructor at (1)
openfast/modules-local/aerodyn/src/AeroDyn_IO.f90:3446:34:

             p%OutParam(I)%Units = ParamUnitsAry(Indx) ! it's a valid output
                                  1
Error: Function 'paramunitsary' at (1) has no IMPLICIT type
```

@ghaymanNREL @bjonkman

**Comment by ghaymanNREL** [comment]
On an Intel VS compile on Windows it issues a warning.  I’ll look in this, maybe there are ‘hidden’ characters, which are not showing up in a visual editor?

**Comment by sayerhs** [comment]
@ghaymanNREL I wonder if this is an ASCII/Unicode issue. One of those characters might be encoded as 2 characters if the file is interpreted as pure ASCII but is has unicode characters.

**Comment by sayerhs** [comment]
Commit b8ea3c23f8ce30 fixes the issue on `GNU gfortran v6.3.0`.

**Comment by bjonkman** [comment]
FYI, those units arrays often contain extended ASCII characters, and I have noticed that GitKracken has a tendency to convert them to unicode if you use it to make other changes to the source file.
